.github/lychee.toml

```diff
@@ -0,0 +1,20 @@
+verbose = "info"
```

Comment:
I found these security related issues. I'll let you review them:

- https://github.com/lycheeverse/lychee/issues/1297
- https://github.com/lycheeverse/lychee/issues/1298

Reply:
🤷‍♂️